src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOidcngResourceServerEntityCommand.php

```diff
@@ -143,18 +144,23 @@ class SaveOidcngResourceServerEntityCommand implements SaveEntityCommandInterfac
      */
     private $manageId;
 
+    /** @var bool */
+    private $isCopy;
+
     public function __construct()
     {
     }
 
     /**
      * @param Service $service
+     * @param bool $isCopy
      * @return SaveOidcngResourceServerEntityCommand
      */
-    public static function forCreateAction(Service $service)
+    public static function forCreateAction(Service $service, bool $isCopy = false)
```

Comment:
And if you are boyscouting, please also set the return type on this method :deciduous_tree:

Reply:
done :-)